src/App.jsx

```diff
@@ -1,70 +1,73 @@
-import 'bulma/css/bulma.css';
-import './App.scss';
+import "bulma/css/bulma.css";
+import "./App.scss";
+import { useState } from "react";
+import classNames from "classnames";
 
 export const goods = [
-  'Dumplings',
-  'Carrot',
-  'Eggs',
-  'Ice cream',
-  'Apple',
-  'Bread',
-  'Fish',
-  'Honey',
-  'Jam',
-  'Garlic',
+  "Dumplings",
+  "Carrot",
+  "Eggs",
+  "Ice cream",
+  "Apple",
+  "Bread",
+  "Fish",
+  "Honey",
+  "Jam",
+  "Garlic",
 ];
 
-export const App = () => (
-  <main className="section container">
-    <h1 className="title is-flex is-align-items-center">No goods selected</h1>
+export const App = () => {
+  const [value, setValue] = useState("Jam");
 
-    <h1 className="title is-flex is-align-items-center">
-      Jam is selected
-      <button data-cy="ClearButton" type="button" className="delete ml-3" />
-    </h1>
+  return (
+    <main className="section container">
+      {value ? (
+        <h1 className="title is-flex is-align-items-center">
+          {value} is selected
+          <button
+            data-cy="ClearButton"
+            type="button"
+            className="delete ml-3"
+            onClick={() => {
+              setValue("");
+            }}
```

Comment:
Create a helper function for this logic and use it here
```suggestion
            onClick={reset}
```